### PR TITLE
Added client to server player orientation communication

### PR DESCRIPTION
**Pull request by NickyW**

### CODE REVIEW

File: Confus/Confus/Networking/ClientConnection.cpp [hunk 1]
```diff
@@ -105,5 +121,5 @@ namespace Confus
 				m_StalledMessages.pop();
 			}
 		}
-    }
+	}
```
Comment (ChristianStiehl) [inline Confus/Confus/Networking/ClientConnection.cpp:124]: Why are all of these lines indented by 1 tab?

File: Confus/Confus/Networking/ClientConnection.h [hunk 2]
```diff
@@ -61,12 +74,12 @@ namespace Confus
...
+			RakNet::SystemAddress getServerAddress() const;
 			/// <summary>
 			/// Dispatches the messages that the connection was not able to send yet
 			/// due to waiting for the connection to be established
 			/// </summary>
 			void dispatchStalledMessages();
-        };
-    }
+		};
+	}
```
Comment (ChristianStiehl) [inline Confus/Confus/Networking/ClientConnection.h:84]: same here?

**Comment by ChristianStiehl** [comment]
Aside from all lines being indented by 1 extra tab, this looks fine to me. Would like to merge this ASAP because I need it to update the other players' rotation + we could use almost the same code for communicating position (would take like 15 mins or less)

**Comment by Athosvk** [comment]
I don't really see why the connection needs to know what an orientation is. The connection should simply send some form of a message. 

Also, wouldn't it be easier if we just passed in a vector 3? If you need to extract the components separately for the packet, you could do that in said method so that we don't have to do that every time we call 'sendOreintation'.

I'm also not too sure about having an entire packet dedicated to orientation, but I guess that's a side-effect of having multiple people working on the start of packet sending

Anyway, fix the identation because it's incredibly hard to read the diff right now.

**Pull request closed without merging**